config/routes.rb

```diff
@@ -115,11 +115,14 @@
   # Quick Reference JSON
   get "/docs/quick-reference/pipelines", to: "quick_reference#pipelines", as: :pipelines_quick_reference
 
+  # Homepage
+  get "/docs/index" => "pages#index", as: :home_page
```

Comment:
Any reason for having the explicit `/docs/index` instead of just `/docs`?

Reply:
Fixed in https://github.com/buildkite/docs/pull/841/commits/45cee9bbee55e9c2cb458eae91b13d8bb46ed24f